src/common/components/suggestion-list/_index.scss

```diff
@@ -143,7 +142,6 @@
         display: flex;
         align-items: center;
         white-space: nowrap;
-        width: 20px;
```

Comment:
@MTSaeed double check all usage of suggestion list component and see that it doesn't break in any usage including mobile view.